docs/advanced-topics/stack-monitoring.asciidoc

```diff
@@ -0,0 +1,160 @@
+:page_id: stack-monitoring
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Stack Monitoring
+
+You can enable link:https://www.elastic.co/guide/en/elasticsearch/reference/current/monitor-elasticsearch-cluster.html[Stack Monitoring]
+on your Elasticsearch and Kibana to collect and ship their metrics and logs.
```

Comment:
```suggestion
on Elasticsearch and Kibana to collect and ship their metrics and logs to a dedicated monitoring cluster.
```
In the ESS docs we emphasize "the dedicated monitoring cluster" so maybe we could use that here too.

Reply:
Sounds good to me.